inscriptions/ISic003469.xml

```diff
@@ -187,87 +192,23 @@
+                    <lb n="3"/><expan><abbr>Ti</abbr><ex>beri</ex></expan> <g ref="#interpunct">·</g> Caesaris <g ref="#interpunct">·</g> Augusti
+                    <lb n="4"/><name>Lutatia</name> <g ref="#interpunct">·</g> <name><expan><abbr>C</abbr><ex>ai</ex></expan></name> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ilia</ex></expan> <g ref="#interpunct">·</g> sacerdos <g ref="#interpunct">·</g> Augustae
+                    <lb n="5"/><add place="overstrike"><expan><abbr>Imp</abbr><ex>eratoris</ex></expan> <expan><abbr>perpet</abbr><ex>ui</ex></expan></add> uxor
+                    <lb n="6"/><name><expan><abbr>M</abbr><ex>arci</ex></expan> <g ref="#interpunct">·</g> Livi <g ref="#interpunct">·</g></name> <name><expan><abbr>M</abbr><ex>arci</ex></expan></name> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ili</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>Qui</abbr><ex>rina</ex></expan> <g ref="#interpunct">·</g> <name>Optati</name> <g ref="#interpunct">·</g> flaminis <g ref="#interpunct">·</g> <expan><ex>Gaul</ex><abbr>itanorum</abbr></expan> <g ref="#interpunct">·</g> <!--for some reason I can't seem to add a supplied reason="lost" for the 'a' within Gaul in the expansion-->
+                    <lb n="7"/>Iuliae <g ref="#interpunct">·</g> Augusti <add place="overstrike"><expan><abbr>Imp</abbr><ex>eratoris</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>perpet</abbr><ex>ui</ex></expan></add> <g ref="#interpunct">·</g> cum <orig>V</orig> <!-- the V here is suggested in some versions, but see EAGLE photos for various reconstructions? -->
+                    <lb n="8"/>liberis <g ref="#interpunct">·</g> <expan><abbr>s</abbr><ex>ua</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>p</abbr><ex>ecunia</ex></expan> consacravit
                 </ab>
-                <!--This EDR text 112580 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <p><!--I have included the overstrike on Imp perpet even though the photos on EAGLE don't appear to show this. Autopsy of stone would help for lines 6 and 8, as there appears to be a restored(?) version of the stone photographed on EAGLE also with a non-restored version?--></p>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#AA">
+                <p>To Ceres Iulia Augusta, mother of Divus Augustus Tiberius Caesar Augustus, Lutatia, daughter of Gaius, sacerdos of Augusta the wife of the perpeptual Imperator, wife of Marcus Livius Optatus, son of Marcus, of tribe Quirina, flamen of the Gaulitani and of Iulia Augusta wife of the perpetual Imperator, consecrated [this] with [her husband?] and children, from her own money</p>
```

Comment:
To Ceres Iulia Augusta (wife of the) deified Augustus, mother of Tiberius Caesar Augustus... [and, further on] ... flamen of the Gaulitani of Iulia Augusta wife of the eternal imperator [the double genitive here means priest (of the people of Gaulus) of Iulia Augusta, so, not priest both of the Gaulitani and Iulia Augusta, but priest of Iulia Augusta, as represenative of / appointed by / on behalf of the Gaulitani]